design/components/card.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { styled } from './stitches.config';
+
+const StyledCard = styled('section', {
+  display: 'inline-block',
+  img: {
+    height: '100%',
+    width: '100%',
+    objectFit: 'cover',
+    transition: `transform .6s ease`,
+  },
+  outline: 'none',
+  position: 'relative',
+
+  'a::after': { position: 'absolute', top: 0, left: 0, right: 0, bottom: 0 },
+
+  '.card-img-container': {
+    height: 324,
+    width: 406,
+    overflow: 'hidden',
+  },
+
+  ':first-child:hover img': {
+    transform: 'scale(1.2)',
+  },
+
+  '.CardContentContainer': {
```

Comment:
Using classes for styling the children that's genius! However I notice these are global class names so we should stick to some convention.

I think BEM is excellent for this.
https://css-tricks.com/bem-101/

We could rename `card-img-container` to `card__image-container` and `CardContentContainer` to `card__content-container`.

Reply:
Great thanks for the link🙂